code/game/objects/structures/crates_lockers/closets/cardboardbox.dm

```diff
@@ -62,3 +64,22 @@
 			for(var/mob/M in viewers(src))
 				M.show_message("<span class='notice'>\The [src] has been cut apart by [user] with \the [WC].</span>", 3, "You hear cutting.", 2)
 			qdel(src)
+			return
+		if(istype(W, /obj/item/weapon/storage/fancy/crayons)) /////addendum by flimflamm
+			var/decalselection = input("Please select a decal") as null|anything in list("atmospherics", "bartender", "barber", "blueshield",	"brig_physician", "captain",
+				"cargo", "chief_engineer",	"chaplain",	"chef", "chemist", "civilian", "clown", "cmo", "coroner", "detective", "engineering", "genetics", "hop",
+				"hos", "hydroponics", "internal_affairs_agent", "janitor",	"magistrate", "mechanic", "medical", "mime", "mining", "nt_representative", "paramedic", "pod_pilot",
+				"prisoner",	"research_director", "security", "syndicate", "therapist", "virology", "warden", "xenobiology")
```

Comment:
Yep that'll do it. When I first started on the replacetext proc i had not yet found the lowertext proc, and so when I tried initially with an actual space and an actual underscore it wouldn't work, which sent me spiraling to find alternative identifiers (thinking space or underscore was not recognizable).

Now that I've got this complete, I think the one remaining finishing touch will be a check to make sure that it is a full box of crayons.